ESP/main/LED.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef LED_HPP
```

Comment:
Make sure clang-format is working, includes  should be in alphabetical order and visibility specifiers should have 2 space indent.